### ISSUE TITLE
annoying pmesh warning

### ISSUE THREAD

**Issue by pmcdonal** [cosmodesi/pypower#6]
Under mpi, I see (np times):

/Users/pmcdonal/opt/anaconda3/lib/python3.7/site-packages/pmesh/domain.py:380: VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this, you must specify 'dtype=object' when creating the ndarray.
  self.edges = numpy.asarray(edges)

I don't know if we can edit pmesh, but can be cleared by:
self.edges = numpy.asarray(edges,dtype=object)

**Comment by adematti** [comment]
Yes! at some point I'd like to revisit the pmesh/pfft-python package, e.g. to allow combined OpenMP / MPI parallelization and use the same package within pyrecon. This is not top priority, though...

**Comment by dforero0896** [comment]
It seems that for more recent versions of numpy this raises an exception and kills the program. I manually added the fix suggested to line 380 in pmesh and everything is fine but it would be a good idea to update the pmesh repo.

**Comment by adematti** [comment]
Hi @dforero0896, I'm using https://github.com/adematti/pmesh
I think it's best to use this one, as it contains a quick fix for https://github.com/rainwoodman/pmesh/issues/48
(in addition to fixing above issue)
